README.md

```diff
@@ -551,4 +551,29 @@ _Jonathan Hersh, Allen Langmaier_
 - **Updates to development**
   - Dynamic updates to list elements
   - `CPListItem`
-  - `CPListImageRowItem`
\ No newline at end of file
+  - `CPListImageRowItem`
+
+## Embrace Swift type inference
```

Comment:
You forgot the TOX entry for this one, could not happen anymore after I merge your prefill-sessions PR, which is awesome btw. 🎉